client/src/components/SearchAddress.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+class SearchAddress extends Component {
+    render() {
+        return (
+            <div className="searchAddress">
+                <h1 className="searchAddress-heading">Does your apartment building need recycling? Build Support!</h1>
+                <div className="searchAddress-inputGroup">
+                    <input type="text" className="searchAddress-input" placeholder="Search for your building"/>
```

Comment:
This is a great start. Just FYI I'm planning to use your branch as a starting place, and then change this input element to google's fancy autosuggest "places" library widget.